README.md

```diff
@@ -54,31 +55,26 @@ TODO Add some context as to why we need this and where it should go.
 <dependency>
     <groupId>com.microsoft.gctoolkit</groupId>
     <artifactId>api</artifactId>
-    <version>2.0.1-SNAPSHOT</version>
+    <version>2.0.1</version>
```

Comment:
I think the version here should refer to the artifact in github packages, not the version in source.